src/annotator/util/shadow-root.js

```diff
@@ -18,7 +22,7 @@ function loadStyles(shadowRoot) {
   // get a usable adder, albeit one that uses browser default styles for the
   // toolbar.
   const styleEl = document.createElement('style');
-  styleEl.textContent = adderStyles.map(url => `@import "${url}";`).join('\n');
+  styleEl.textContent = `@import "${url}";`;
```

Comment:
In both the old and new code we were not escaping the URL in any way. That is currently safe because `HTMLLinkElement.href` returns an escaped URL where any quotes are URL-escaped as `%22`.